### PR TITLE
Docker setup |  Custom express server

### PR DESCRIPTION
**Pull request by sunnil24**

### CODE REVIEW

File: templates/base/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "export": "next export"
   },
   "dependencies": {
+    "express": "^4.17.1",
```
Comment (vinodloha) [inline templates/base/package.json:12]: Lets discuss this again tomorrow, where optional dependencies should be added?

File: templates/ssr/customServer.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+COPY package.json .
+RUN yarn install
+COPY . .
+RUN yarn build && yarn --production
+
+FROM node:alpine
+WORKDIR /app
+COPY --from=builder /app .
+EXPOSE 3000
+CMD ["yarn", "run", "dev"]
```
Comment (vinodloha) [inline templates/ssr/customServer.Dockerfile:12]: Docker would be starting prod build

File: templates/ssr/customServer.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FROM node:alpine AS builder
```
Comment (vinodloha) [inline templates/ssr/customServer.Dockerfile:1]: We also discussed that there will be 3 options now 

1. static export
2. SSR default 
3. SSR custom

File: templates/ssr/package.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "scripts": {
+    "dev": "node server/index"
```
Comment (vinodloha) [inline templates/ssr/package.json:3]: should be start

File: templates/ssr/customServer.Dockerfile (hunk 2 above)
Comment (sunnil24) [reply templates/ssr/customServer.Dockerfile]: Done

File: templates/ssr/package.json (hunk 4 above)
Comment (sunnil24) [reply templates/ssr/package.json]: Done

**Pull request merged**